README.md

```diff
@@ -46,7 +46,8 @@ framework:
 
 ### SQS
 
-The [SQS](https://aws.amazon.com/sqs/) service is a queue that works similar to RabbitMQ. To use it, create a SQS queue and set its URL as the DSN:
+The [SQS](https://aws.amazon.com/sqs/) service is a queue that works similar to RabbitMQ. To use it, set its URL in the 
+envrionment variable `MESSENGER_TRANSPORT_DSN`:
```

Comment:
```suggestion
The [SQS](https://aws.amazon.com/sqs/) service is a queue that works similar to RabbitMQ. To use it, set its URL in the environment variable `MESSENGER_TRANSPORT_DSN`:
```

Typo (+ I'd rather avoid line breaks in sentences, it simplifies diffs, git blame, github suggestions, etc.)